src/windows/main_window.py

```diff
@@ -1227,13 +1227,29 @@ def actionAddMarker_trigger(self, event):
         marker.data = {"position": position, "icon": "blue.png"}
         marker.save()
 
-    def actionPreviousMarker_trigger(self, event):
-        log.info("actionPreviousMarker_trigger")
 
-        # Calculate current position (in seconds)
+    def findAllMarkerPositions(self):
+        """Build and return a list of all seekable locations for the currently-selected timeline elements"""
+
+        def getKeyframePositions(data, clip_start_time, clip_stop_time, fps_float):
+            """ Add all keyframes of a clip.data to all_marker_positions """
```

Comment:
I wouldn't use "clip" here. It could confuse someone reading the code in the future, since it's actually called for both clips and transitions. Something like "timeline element" is suitably generic.